tests/CMakeLists.txt

```diff
@@ -10,8 +10,7 @@ endif()
 enable_testing()
 #needs to support -m32 since the memory tests assume a 32 bit architecture
 if (UNIX AND NOT APPLE)
-    add_compile_options(-m32 -Og -ggdb3)
-    add_link_options(-m32)
+    add_compile_options(-Og -ggdb3)
     add_subdirectory(32bit_unit_tests)
```

Comment:
You can just keep this out of the if Block and delete the rest - it'll let you run these tests locally on non Linux systems while developing (./dbt test)